site/app/controllers/pdf/PDFController.php

```diff
@@ -71,6 +71,9 @@ private function showStudentPDF(){
                 }
             }
         }
+        $ta_graded_gradeable = $graded_gradeable->getTaGradedGradeable();
```

Comment:
The result of `getTaGradedGradeable` can be null, so you should check that.  Also noticed that there are a lot of unchecked results above in the function.  Don't need to change those for this PR, but something you might want to more gracefully handle. (i.e. `tryGetGradeable` will return `false` on failure)